Recrutify/Recrutify.DataAccess/Models/ProjectResultAssigned.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Recrutify.DataAccess.Models
+{
+    public class ProjectResultAssigned
```

Comment:
AssignedCandidateProjectResult, 
also use inheritance with ProjectResult, not to duplicate

Reply:
done